nbgrader/exchange/exchange.py

```diff
@@ -125,3 +125,34 @@ def start(self):
         self.init_src()
         self.init_dest()
         self.copy_files()
+
+    def get_current_user_courses(self):
+        """Check if student is enrolled in course"""
+        from tornado.httpclient import HTTPClient, HTTPRequest
+
+        if os.getenv('JUPYTERHUB_API_TOKEN'):
```

Comment:
So just to clarify, this is an environment variable that's automatically set by JupyterHub, even in new terminal environments? So if I launch a terminal from the notebook server and try to run `nbgrader list`, will that still work appropriately?

Reply:
yes, correct, but just on Jupyterhub >= 0.8, this feature of `nbgrader` would **not** work on 0.7.2

Reply:
Ok, good to know, we should probably mention that in the docs too.